### ISSUE TITLE
Tamper Prevention

### ISSUE THREAD

**Issue by zizii1** [wazuh/wazuh#23470]
Hello, we would really appreciate if Wazuh relase a tamper-prevention feature to protect the logs from accidental or intentional delete by unauthorised actors.
To give more context, in the IT/ Security teams , that have access to dashboard , one engineer could be an insider attacker, we want to ensure a tracebility and implement a tamper prevention feature.
Let me know your thoughts.

**Comment by kclinden** [comment]
Agree. Many commercial products provide some sort of advanced protection from a threat actor disabling an EDR agent. If a system is compromised with elevated privileges they should not be able to disable the wazuh agent easily. I have seen some that require a password to disable even if you are an administrator on the box.
